scripts/commit_classifier.py

```diff
@@ -381,7 +381,7 @@ def load_user(phid):
 
             if self.git_repo_dir:
                 patch_proc = subprocess.Popen(
-                    ["patch", "-p1", "--no-backup-if-mismatch", "--force"],
+                    ["git", "cinnabar", "hg2git", patch.phid],
```

Comment:
`patch.phid` is not a changeset, it is a Phabricator ID. 

The command is searching for a git commit that is corresponding to the provided hg changeset. This is not what we want to do. The goal here is to fetch the changeset from the local Mercurial repository. See https://github.com/glandium/git-cinnabar#fetching-a-specific-mercurial-changeset.